diy_asylum_frontend/src/components/LandingPage/index.js

```diff
@@ -0,0 +1,166 @@
+import React, { Component } from "react";
+import "./landing.css";
+
+const LandingPage = () => (
+  <div>
+    <body>
```

Comment:
This `<body`> tag is throwing an error for me in the console.
To see the devtool console, just open this page in chrome and press `Command+Option+I` and make sure the `console` tab is selected.

You should see an error at the top that says ```
validateDOMNesting(...): <body> cannot appear as a child of <div>```

There are also some other warnings and errors in there that would be great if you could comb thru and sort out!